discovery/uyuni/uyuni.go

```diff
@@ -68,6 +68,7 @@ type systemGroupID struct {
 type networkInfo struct {
 	SystemID int    `xmlrpc:"system_id"`
 	Hostname string `xmlrpc:"hostname"`
+	FQDN     string `xmlrpc:"fqdn"`
```

Comment:
@witekest just be aware that the name of this field is still under discussion and may change in the near future (it was recently changed from `fqdn` to `primary_fqdn`).

See https://github.com/uyuni-project/uyuni/pull/3170#discussion_r576172366 if you want to contribute to the discussion.

Reply:
Changed to `primary_fqdn` as well.